vignettes/Affymetrix.Rmd

```diff
@@ -1,71 +1,61 @@
-%\VignetteIndexEntry{Preparing Affymetrix Data}
-%\VignetteDepends{GWASTools, GWASdata}
-
-\documentclass[11pt]{article}
-\usepackage{fullpage}
-\usepackage{Sweave}
-\usepackage{amsmath}  
-\usepackage{graphicx}
-\usepackage[utf8]{inputenc}
-\usepackage[pdftex,plainpages=false, letterpaper, bookmarks, bookmarksnumbered, colorlinks, linkcolor=blue, citecolor=blue, filecolor=blue, urlcolor=blue]{hyperref}
-
-% Different font in captions
-\newcommand{\captionfonts}{\footnotesize}
-
-\begin{document}
-
-\title{Preparing Affymetrix Data}
-\author{GENEVA Coordinating Center\\
-  Department of Biostatistics\\
-  University of Washington}
-
-\maketitle
-
-\SweaveOpts{keep.source=TRUE, eps=FALSE} 
-
-
-\section{Overview}
-This vignette describes how to prepare Affymetrix data for use in
-GWASTools.  After following the examples here for preparing the data,
-the analysis procedure is the same as presented in
-the GWAS Data Cleaning vignette.  For a description of the file
-formats (NetCDF and GDS), see the Data Formats vignette.
-
-
-\section{Creating the SNP Annotation Data Object}
-\label{create_snpann}
-
-All of the functions in GWASTools require a minimum set of variables in the SNP annotation data object.  The minimum required variables are
-
-\begin{itemize}
-\item{{\tt snpID}, a unique integer identifier for each SNP}
-\item{{\tt chromosome}, an integer mapping for each chromosome, with values 1-27, mapped in order from 1-22, 23=X, 24=XY (the pseudoautosomal region), 25=Y, 26=M (the mitochondrial probes), and 27=U (probes with unknown positions)}
-\item{{\tt position}, the base position of each SNP on the chromosome.}
-\end{itemize}
-
-
-We create the integer chromosome mapping for a few reasons.  The
-chromosome is stored as an integer in the NetCDF or GDS files, so in order to
-link the SNP annotation with the NetCDF/GDS file, we use the integer
-values in the annotation as well.
-For convenience when using GWASTools functions,
-the chromosome variable is most times assumed to be an integer value.  Thus, for the sex chromosomes, we can simply
-use the {\tt chromosome} values.  For presentation of results, it is important to have the mapping of the
-integer values back to the standard designations for the chromosome
-names, thus the {\tt getChromosome()} functions in the GWASTools
-objects have a {\tt char=TRUE} option to return the characters 1-22,
-X, XY, Y, M, U.
-The position variable should hold all numeric values of the physical position of a probe.  {\em The SNP annotation file is assumed to list the probes in order of chromosome and 
-position within chromosome.}
-
-Note that for Affymetrix data, the rs ID is often not unique, as there
-may be multiple probes for a given SNP.  The probe ID is usually the
-unique SNP identifier.
-Also, Affymetrix annotation generally has a separate column or columns
-to indicate pseudoautosomal regions; some manipulation is usually
+---
+title: "Preparing Affymetrix Data"
+author:
+  - name: "GENEVA"
```

Comment:
Please change to:
name: "GENEVA Coordinating Center"
affiliation: "Department of Biostatistics, University of Washington"